.gitignore

```diff
@@ -14,8 +14,8 @@ bin
 *.out
 
 # Kubernetes Generated files - skip generated files, except for vendored files
-
 !vendor/**/zz_generated.*
+**/zz_generated.*
```

Comment:
Single .gitignore addition to ignore all `zz_generated` files in project, unless in vendor.